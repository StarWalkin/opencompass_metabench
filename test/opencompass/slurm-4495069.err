  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [38:37<00:00, 2317.16s/it]100%|██████████| 1/1 [38:37<00:00, 2317.17s/it]
  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:11<00:00, 11.92s/it]100%|██████████| 1/1 [00:11<00:00, 11.92s/it]
