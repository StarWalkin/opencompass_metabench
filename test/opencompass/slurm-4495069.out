/u/zzhang27/.conda/envs/opencompass/bin/python3
08/16 19:30:48 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/16 19:30:48 - OpenCompass - INFO - Loading hf_qwen2_7b_instruct: configs/models/qwen/hf_qwen2_7b_instruct.py
08/16 19:30:48 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/16 19:30:49 - OpenCompass - INFO - Current exp folder: outputs/default/20240816_193049
08/16 19:30:49 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/16 19:30:49 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLInfer[qwen2-7b-instruct-hf/gsm8k] on GPU 0
08/16 20:09:27 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLEval[qwen2-7b-instruct-hf/gsm8k] on CPU 
dataset    version    metric    mode      qwen2-7b-instruct-hf
---------  ---------  --------  ------  ----------------------
gsm8k      1d7fe4     accuracy  gen                      81.35
08/16 20:09:39 - OpenCompass - INFO - write summary to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240816_193049/summary/summary_20240816_193049.txt
08/16 20:09:39 - OpenCompass - INFO - write csv to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240816_193049/summary/summary_20240816_193049.csv
All processes have been started and their PIDs stored in process_ids_gpt4o.txt.
All processes have completed.
